manifests/j/jumppad/jumppad/0.5.57/jumppad.jumppad.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: jumppad.jumppad
+PackageVersion: 0.5.57
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: jumppad.exe
+  PortableCommandAlias: jumppad

```

```suggestion
  PortableCommandAlias: jumppad
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Docker.DockerDesktop
    - PackageIdentifier: RedHat.Podman
    - PackageIdentifier: Git.Git
```
If another version of Git is preferred, feel free to substitute it.

---

Even with these preinstalled, the application errors.
```PowerShell
PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Links\jumppad.exe

###### SYSTEM DIAGNOSTICS ######
 [  ERROR  ] Docker
 [  ERROR  ] Podman
 [   OK    ] Git


* Unable to connect to Docker, ensure Docker is installed and running.
* Unable to connect to Podman, ensure Podman is installed and running.

PS C:\Users\User\Desktop> &"C:\Program Files\RedHat\Podman\podman.exe"
```